lib/mix/tasks/update_protobufs.ex

```diff
@@ -7,7 +7,9 @@ defmodule Mix.Tasks.GenerateTestProtobufs do
   @shortdoc "Generates test protobufs"
   def run(_arg) do
     run_system_command(
-      'protoc --proto_path=test/support/ipc/protobuf --elixir_out=test/support/ipc/messages test/support/ipc/protobuf/*.proto'
+      'protoc --proto_path=test/support/ipc/protobuf
+              --elixir_out=test/support/ipc/messages
+              test/support/ipc/protobuf/*.proto'
     )
```

Comment:
Credo was complaining the line was too line. Had to look up how to do multi-line strings in Elixir... this was the suggestion. Not sure if it's idiomatic.

Reply:
I think something like this would be better:
```suggestion
"""
protoc --proto_path=test/support/ipc/protobuf \
  --elixir_out=test/support/ipc/messages \
  test/support/ipc/protobuf/*.proto
"""
```
Single quotes and double quotes are pretty different in elixir so I'm actually surprised the singles worked here. 
I added the `\` for new line as I would expect a multi line system command would need.

Reply:
Yep, I like your version better.